### DISCUSSION TITLE
SBOL visualization

### DISCUSSION THREAD

**Discussion by kenza12** [SynBioDex/SBOL-utilities#184] [General]
Hi @jakebeal ,

I'm trying to integrate `sbol-converter` to Galaxy. And I was wondering if you're using a specific visualizer for **sbol3**. I tested `SBOLCanvas` and `Visbol`, trying to load an sbol3, using your test output files (converted to xml) but it didn't work. It seems that SBOLCanvas supports only sbol2.
Thank you.

**Comment by jakebeal** [comment]
While there is not currently a visualizer that ingests SBOL3 files, constructs in SBOL3 can be visualized in two ways:
1) converting to SBOL2 and using visbol, or
2) mapping into graphics yourself with the aid of dnaplotlib

**Reply by kenza12** [reply]
Ok cool, thanks for the information !
